src/ILLink.Shared/DiagnosticString.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Resources;
+
+namespace ILLink.Shared
+{
+	public readonly struct DiagnosticString
+	{
+		readonly string _titleFormat;
+		readonly string _messageFormat;
+		static readonly ResourceManager _resourceManager = SharedStrings.ResourceManager;
```

Comment:
Why do we need a static field here? It seems like a local variable would suffice